source/designers-guide/how-to-use-karma/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: How to use Karma for storefront tests
+github_link: designers-guide/how-to-use-karma/index.md
+shopware_version: 5.0.3
+indexed: true
+---
+
+## Introducing
+[Karma](http://karma-runner.github.io/) is a test runner for JavaScript tests for the storefront of Shopware. The tests itself are written in [Jasmine](http://jasmine.github.io/).
+
+The main goal for Karma is to bring a productive testing environment to developers. The environment being one where they don't have to set up loads of configurations, but rather a place where developers can just write the code and get instant feedback from their tests.
+
+## Requirements 
+Karma is based on Node.js, which makes it necessary to have [Node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/) installed on your system. Node.js is available on a majority of systems and distrubition. If your system isn't listed below, please use the [official Node.js installation guide](https://github.com/nodesource/distributions).
```

Comment:
Karma requires [Node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/) to be installed on your system. Node.js is available on a majority of systems and distributions. If your system isn't listed below, please use the [official Node.js installation guide](https://github.com/nodesource/distributions).